sdk/servicebus/service-bus/src/serviceBusAtomManagementClient.ts

```diff
@@ -156,7 +156,7 @@ export interface RuleResponse extends RuleProperties, Response {}
  * These objects also have a property called `_response` that you can use if you want to
  * access the direct response from the service.
  */
-export class ServiceBusManagementClient extends ServiceClient {
+export class ServiceBusAdministrationClient extends ServiceClient {
```

Comment:
Maybe.. It's time to rename this file too.

Reply:
I would have it was just "management", but since it is "AtomManagement", I will leave it as is. Does not affect end user